rnmodules/react-native-kb/ios/Kb.mm

```diff
@@ -22,6 +22,24 @@
 using namespace std;
 using namespace kb;
 
+// used to keep track of objects getting destroyed on the js side
```

Comment:
this is all shifted around so helpers are up top and the exposed methods are below